[cppgraphqlgen] build failure

**Host Environment**
- OS: Linux
- Compiler: 
gernot@Io:~/Development/vcpkg$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/8/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 8.3.0-26ubuntu1~18.04' --with-bugurl=file:///usr/share/doc/gcc-8/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++ --prefix=/usr --with-gcc-major-version-only --program-suffix=-8 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-libmpx --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 8.3.0 (Ubuntu 8.3.0-26ubuntu1~18.04) 

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install cppgraphqlgen`

**Failure logs** 
gernot@Io:~/Development/vcpkg$ ./vcpkg install cppgraphqlgen
Computing installation plan...
The following packages will be built and installed:
    cppgraphqlgen[core]:x64-linux
Starting package 1/1: cppgraphqlgen:x64-linux
Building package cppgraphqlgen[core]:x64-linux...
-- Using cached /home/gernot/Development/vcpkg/downloads/microsoft-cppgraphqlgen-v3.1.1.tar.gz
-- Using source at /home/gernot/Development/vcpkg/buildtrees/cppgraphqlgen/src/v3.1.1-f2eca4fdc5
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:136 (message):
    Command failed: /home/gernot/Development/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/bin/cmake --build . --config Debug --target install -- -v
    Working Directory: /home/gernot/Development/vcpkg/buildtrees/cppgraphqlgen/x64-linux-dbg
    See logs for more information:
      /home/gernot/Development/vcpkg/buildtrees/cppgraphqlgen/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:91 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/cppgraphqlgen/portfile.cmake:17 (vcpkg_install_cmake)
  scripts/ports.cmake:90 (include)


Error: Building package cppgraphqlgen:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: cppgraphqlgen:x64-linux
  Vcpkg version: 2020.02.04-unknownhash

Additionally, attach any relevant sections from the log files above.
gernot@Io:~/Development/vcpkg$ ./vcpkg update
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.
gernot@Io:~/Development/vcpkg$

--- a/ports/cppgraphqlgen/CONTROL
+++ b/ports/cppgraphqlgen/CONTROL
@@ -1,5 +1,5 @@
 Source: cppgraphqlgen
-Version: 3.1.1
+Version: 3.2.0
 Build-Depends: boost-filesystem (!uwp&!windows), boost-program-options, pegtl, rapidjson
 Description: C++ GraphQL schema service generator
 Homepage: https://github.com/microsoft/cppgraphqlgen

--- a/ports/cppgraphqlgen/portfile.cmake
+++ b/ports/cppgraphqlgen/portfile.cmake
@@ -22,4 +22,6 @@ vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/cppgraphqlgen)
 
 vcpkg_copy_pdbs()
 
-configure_file(${SOURCE_PATH}/LICENSE ${CURRENT_PACKAGES_DIR}/share/cppgraphqlgen/copyright COPYONLY)
+file(INSTALL ${SOURCE_PATH}/LICENSE
+    DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
+    RENAME copyright)

--- a/ports/cppgraphqlgen/portfile.cmake
+++ b/ports/cppgraphqlgen/portfile.cmake
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO microsoft/cppgraphqlgen
-    REF v3.1.1
-    SHA512 c6ebffcfd6f563b2a4ba1822ca173c73fd41b6f1dfbd4f94e64bee4bff626c64d88b2a35fa4c38189ba16852945f53d1f60dd68dc0ea85eb036ec31c94e7bdf4
+    REF v3.2.0
+    SHA512 21b0ea4b7bcdfa8107018e3a53413f7242dfd3fe5f66c4c7f2b351b5aa7227281eb78b47462fae3e0ba203ad028e98d6d0ec6bbd93c90be56ab657007953e522
     HEAD_REF master
 )
 

--- a/ports/cppgraphqlgen/portfile.cmake
+++ b/ports/cppgraphqlgen/portfile.cmake
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO microsoft/cppgraphqlgen
-    REF v3.2.0
-    SHA512 21b0ea4b7bcdfa8107018e3a53413f7242dfd3fe5f66c4c7f2b351b5aa7227281eb78b47462fae3e0ba203ad028e98d6d0ec6bbd93c90be56ab657007953e522
+    REF 82763064a0551be86fa63f320206e863beef47c0 # v3.2.0
+    SHA512 7c88cf95c0aa09eed18c8a807e4ee3ae7a74958e0bb95714f6e725d54bf4fadbce5438150e653d142aadcbeb2f167d5faad94b90728fd13c3fe54730fec559f0
     HEAD_REF master
 )
 

